contracts/ColuLocalCurrency.sol

```diff
@@ -12,21 +12,33 @@ contract ColuLocalCurrency is Ownable, Standard677Token, TokenHolder {
     string public name;
     string public symbol;
     uint8 public decimals;
-   
+    string public tokenURI;
+
+    event TokenURIChanged(string tokenURI);
```

Comment:
Change name `tokenURI` to `newTokenURI`
Although the name of the param is not part of the event signature, it is better for understanding the event.